### PR TITLE
Add license field to package.json

### PR DESCRIPTION
**Pull request by SimenB**
I just ran `npm init`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,5 +92,14 @@
...
+  ],
+  "bugs": {
+    "url": "https://github.com/browsersync/browser-sync/issues"
+  },
+  "main": "index.js",
+  "directories": {
+    "example": "examples",
+    "test": "test"
+  },
+  "license": "Apache-2.0"
```
Comment (shinnn) [inline package.json]: Not needed. https://github.com/BrowserSync/browser-sync/blob/913b51ea5c0c7f3a481c229c00b8d1eecea63471/package.json#L10-L15

File: package.json [hunk 2]
```diff
@@ -92,5 +92,14 @@
     "css",
     "live reload",
     "sync"
-  ]
+  ],
+  "bugs": {
+    "url": "https://github.com/browsersync/browser-sync/issues"
+  },
+  "main": "index.js",
```
Comment (shinnn) [inline package.json]: Not needed.

Comment (SimenB) [inline package.json]: `npm init` added it. no problem to remove though

File: package.json [hunk 3]
```diff
@@ -92,5 +92,14 @@
     "css",
     "live reload",
     "sync"
-  ]
+  ],
+  "bugs": {
```
Comment (shinnn) [inline package.json]: The `repository` field already indicates this repository.

File: package.json (hunk 1 above)
Comment (SimenB) [inline package.json]: Array way of declaring it is deprecated.

https://github.com/npm/npm/pull/8179

I did not notice the array though, I'll fix it later today

File: package.json (hunk 3 above)
Comment (SimenB) [inline package.json]: Again, `npm init` added it automatically

**Comment by shakyShane** [comment]
@shinnn - anything here we need? if not, close.

@SimenB - thanks for contributing.

**Comment by SimenB** [comment]
@shakyShane As mentioned, they way you are now declaring license is deprecated in current `npm@next`. I'll fix this PR in a couple of hours.

**Comment by shinnn** [comment]
@shakyShane The only one change we should pick up is (as @SimenB said) updating `licenses` to a    simpler `license`.

All the other changes are not needed.

**Comment by SimenB** [comment]
@shinnn Updated now :smile:

**Comment by shinnn** [comment]
:+1: Thanks, @SimenB.

**Pull request merged**